.github/workflows/release.yaml

```diff
@@ -1,8 +1,6 @@
 name: 'Publish to npm'
 on:
-  push:
-    tags:
-      - 'v*'
+  workflow_call:
```

Comment:
Publishing to npm based on a workflow call. If we want to still publish on tag we can add workflow_dispatch to call this manually